components/director/internal/domain/scenarioassignment/service_test.go

```diff
@@ -15,9 +17,10 @@ func TestService_Create(t *testing.T) {
 	t.Run("happy path", func(t *testing.T) {
 		// GIVEN
 		mockRepo := &automock.Repository{}
-		defer mockRepo.AssertExpectations(t)
 		mockRepo.On("Create", mock.Anything, fixModel()).Return(nil)
-		sut := scenarioassignment.NewService(mockRepo)
+		mockScenarioDefSvc := mockScenarioDefServiceThatReturns([]string{"scenario-A"})
```

Comment:
After rebase you could use `scenarioName` const here.
Also, please check other places where you could use `tenantID` and `errMsg` consts.